gulpfile.js

```diff
@@ -1,87 +1,22 @@
-// Include Gulp and Plugins
-var _           = require('underscore');
-var argv        = require('yargs').argv;
-var colors      = require('colors');
-var fm          = require('front-matter');
-var fs          = require('fs');
-var gulp        = require('gulp');
-var gzip        = require('gulp-gzip');
-var handlebars  = require('handlebars');
-var marked      = require('marked');
-var mkdirp      = require('mkdirp');
-var moment      = require('moment');
-var rename      = require('gulp-rename');
-var requireDir  = require('require-dir');
-var rmrf        = require('rimraf');
-var sass        = require('gulp-sass');
+'use strict';
 
-var conf        = require('./conf.json');
-var dir         = requireDir('./tasks/');
+// Include Gulp and modular task file
+var gulp = require('gulp');
+var tasks = require('./lib/tasks/');
+var t = title || null, 
+    s = slug ||  null;
```

Comment:
Same as comment for `title`, please remove and put `slug in the appropriate gulp task